components/Card.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 import Image from "next/image";
-import deleteIcon from "../public/images/deleteIcon.svg";
+import deleteIcon from "../public/images/deleteIcon.png";
```

Comment:
SVG is usually better than PNG. Why did you change that?

Reply:
> SVG is usually better than PNG. Why did you change that?

svg loading was damn slow in inspect modus.